sklearn/utils/extmath.py

```diff
@@ -696,9 +696,9 @@ def _batch_mean_variance_update(X, old_mean, old_variance, old_sample_count):
 
 def _deterministic_vector_sign_flip(u):
```

Comment:
Can we use this function inside `svd_flip`? Is there any reason why `svd_flip` uses the max instead of the first element? 

I will try to dig up the original source of inspiration from when I initially wrote the first iteration of `svd_flip`.